lib/node_modules/@stdlib/assert/is-alphanumeric/test/test.js

```diff
@@ -0,0 +1,125 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2018 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MODULES //
+
+var tape = require( 'tape' );
+var isAlphaNumeric = require( './../lib' );
+
+
+// TESTS //
+
+tape( 'main export is a function', function test( t ) {
+	t.ok( true, __filename );
+	t.strictEqual( typeof isAlphaNumeric, 'function', 'main export is a function' );
+	t.end();
+});
+
+tape( 'the function returns `true` if provided a string containing all numeric digits', function test( t ) {
+	t.strictEqual( isAlphaNumeric( '0123456789' ), true, 'returns true' );
+	t.end();
+});
+
+tape( 'the function returns `true` if provided a string containing all alphanumeric characters', function test( t ) {
+	var values;
+	var i;
+
+	values = [
+		'a123',
+		'b',
+		'c',
+		'd',
+		'e',
+		'f456',
+		'123g',
+		'hacd',
+		'idc',
+		'54',
+		'j21',
+		'kasd',
+		'l',
+		'm',
+		'n',
+		'o',
+		'pop',
+		'q234',
+		'r98765',
+		's',
+		't',
+		'u',
+		'v',
+		'w1234123',
+		'sdcasdx',
+		'354367854y',
+		'z',
+		'6583924A',
+		'BFUNFPIO443',
+		'C',
+		'DRCJENRJNRFWERF',
+		'E',
+		'F',
+		'G',
+		'H',
+		'I',
+		'J',
+		'K',
+		'L',
+		'M',
+		'N',
+		'O',
+		'P',
+		'Q',
+		'R',
+		'S',
+		'T',
+		'U',
+		'V',
+		'W',
+		'X',
+		'Y',
+		'Z'
+	];
+
+	for ( i = 0; i < values.length; i++ ) {
+		t.strictEqual( isAlphaNumeric( values[i] ), true, 'returns true when provided ' + values[i] );
+	}
+	t.end();
+});
+
+tape( 'the function returns `false` if provided a string not containing exclusively alphanumeric characters', function test( t ) {
```

Comment:
Test description does not match what is tested, as the description makes it sound like only strings are passed in. Maybe rephrase to `if not provided a string containing exclusively alphanumeric characters`.

Reply:
Resolved!